java/lang/Double/DoubleTest.cpp

```diff
@@ -268,6 +268,22 @@ TEST (JavaLang, DoubleValueOf) {
     Double::compare(expectedResult.doubleValue(), actualResult.doubleValue());
 }
 
+TEST (JavaLang, DoubleValueOfWithDoubleInput) {
```

Comment:
Please tell me why not use `ASSERT` in this test case. 
Does `Double::compare(expectedResult.doubleValue(), actualResult.doubleValue());` show fail when we run unit test?